dev/PowerNotifications/PowerNotifications.h

```diff
@@ -467,22 +566,26 @@ namespace winrt::Microsoft::Windows::System::Power
             // DisplayStatus Functions
             Power::DisplayStatus DisplayStatus()
             {
+                auto activity = PowerNotifications::DisplayStatusUpdateActivity::Start();
```

Comment:
Are there perf implications to all these WIL activities? How often do these fire, and how much work happens? "Lots" and "little" would make the activity overhead significant.

Reply:
I have added an Activity for every API call. ie for every Power API set like PowerStatus an activity for register, unregister, update value and callback. I haven't done telemetry before so don't know if this is enough/more than needed. @andreww-msft thoughts?
@ZenBird could you answer the perf query?

Reply:
activities are indeed expensive. I would rather log events with HR and wrap the functions in a try-catch block and log the failing hr from there.

Reply:
Re activities: yes that seems like overkill. ZenBird's suggestion makes sense to reduce the perf hit.